resources/views/includes/sidebar.blade.php

```diff
@@ -9,7 +9,8 @@
                 </a>
             </li>
             <li>
-                <a href="">
+                <!-- redirects to profile page based on username -->
+                <a href="profile/{{ Auth::user()->username }}">
```

Comment:
please note @SplitSet3R/front-end that the routes should start with "./" 

I will demo tomorrow in our meeting what could be happen if this isnt followed